### ISSUE TITLE
skipping NVT 'x.x.x...' from import of config 'x-x-x...' because the NVT does not have a family

### ISSUE THREAD

**Issue by jfdarveau** [Atomicorp/gvm#50]
Hi,

Thanks for providing GVM packages for CentOS/RHEL. It's very easy to deploy and convenient to maintain.

I've got an issue with config import after a fresh installation. Apparently, it was fixed in 20.8.1. Is it possible to upgrade the packages?

https://community.greenbone.net/t/skipping-nvt-1-3-6-1-4-1-25623-1-0-150081/8280
https://github.com/greenbone/gvmd/pull/1366

Thank you

**Comment by mikch06** [comment]
I was informed that they will update the packages last week. Waiting on a package update since an issue in november.

**Comment by atomicturtle** [comment]
I appreciate everyones patience on this, its a very very complicated build chain and takes  weeks to assemble correctly.  If you'd like to help, you can join us on the OSSEC slack.  https://join.slack.com/t/ossec/shared_invite/zt-mejtq1hy-z_RfsHgaWu0C9pq1aiwSTg

**Comment by atomicturtle** [comment]
Well this was pretty out of date. I reckon it was resolved with 21.4.0. And 21.4.3 is out, but please open a new issue here if you run into this again
